### ISSUE TITLE
[REQUEST] Automate Aircraft Bug Report / Feature request

### ISSUE THREAD

**Issue by MisterChocker** [flybywiresim/installer#248]
**Is your feature request related to a problem? Please describe.**

For some users it may be hard or annoying to always "search" for the build_info.json

**Describe the feature you'd like to see implemented**

_Since we are using the beta template feature, we are able to automatically prefill forms._

A dedicated button to directly report a bug for the corresponding aircraft which prefills the form the build_info.json content as in the example below.

I'd like to see a dedicated button on the aircrafts tab which is easy to find and use to make the (bug report) process faster and easier for the user.

e.g. this [link](https://github.com/flybywiresim/a32nx/issues/new?labels=Bug&template=bug_report_beta.yaml&build-info=%7B%0A%20%20%22built%22%3A%20%222021-04-15T22%3A57%3A32%2B00%3A00%22%2C%0A%20%20%22ref%22%3A%20%22v0.6.0%22%2C%0A%20%20%22sha%22%3A%20%22981b2e4b0e89998d21a70fe26de54a2f10066f6e%22%2C%0A%20%20%22actor%22%3A%20%22Benjozork%22%2C%0A%20%20%22event_name%22%3A%20%22manual%22%0A%7D) prefills the build_info.json with this content of the build_info.json and the user doesn't need to manually search for the build_info.json when reporting issues via the installer.
```json
{
  "built": "2021-04-29T14:32:22+00:00",
  "ref": "HEAD",
  "sha": "e049d433cbb5d8c53ce083a4631626309c13c86a",
  "actor": "David Walschots <davidwalschots@users.noreply.github.com>",
  "event_name": "manual"
}

```

**Additional context**

Discord username (if different from GitHub): DerL30N#3751

**Comment by FoxtrotSierra6829** [comment]
We already have a report button
![grafik](https://user-images.githubusercontent.com/50082450/116783654-2c4a2000-aa90-11eb-811d-273024b526ad.png)

**Comment by MisterChocker** [comment]
Yes, this does not auto populate the build info field on the template. It just directs to the selection tab and it also does not differ between e.g. aircrafts etc

**Comment by FoxtrotSierra6829** [comment]
Yes. I know, but as said, we already have the button, so no need for another one IMO. Just improve this one.

**Comment by MisterChocker** [comment]
Especially when looking towards multiple aircraft supported by the installer, I believe a solution which allows each aircraft to have it's own report button(s) which points at each individual (e.g. github report template) will be extremely useful.

**Comment by FoxtrotSierra6829** [comment]
I didn't say anything against your points. Just that we already have a button.

**Comment by MisterChocker** [comment]
We can use that button to direct to the installer. (at least what I would've expect that button to do)

**Comment by AdamNZ** [comment]
> I didn't say anything against your points. Just that we already have a button.

Where is this button, exactly? I don't see it in my installer/updater.

**Comment by FoxtrotSierra6829** [comment]
> 
> 
> > I didn't say anything against your points. Just that we already have a button.
> 
> Where is this button, exactly? I don't see it in my installer/updater.

Directly left of the minimize window button
